src/core/kernel/symbol.rs

```diff
@@ -187,8 +187,23 @@ impl fmt::Display for Symbol {
     }
 }
 
+pub(crate) fn matching_events_to_symbols(target: &str) -> Result<Vec<Symbol>> {
+    let symbols = inspector()?
+        .kernel
+        .matching_events(target)?
+        .iter()
+        .filter_map(|t| Symbol::from_name(t).ok())
+        .collect::<Vec<Symbol>>();
+
+    if symbols.is_empty() {
+        bail!("Could not get traceable symbols matching '{target}'");
```

Comment:
Nit. It'd be nice if the error says it's a tracepoint, something like s/traceable symbols/tracepoint/  and probably s/get/find/